src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -238,6 +238,7 @@ public AdjudicationDetail getAdjudication(@PathVariable("offenderNo") @ApiParam(
                     nvl(pageLimit, 10L));
 
             return ResponseEntity.ok()
+                    .headers(pagedCaseNotes.getPaginationHeaders())
```

Comment:
Typo? (duplicated below)

Reply:
Ah, nice spot. I think it was a merge fail.

Reply:
Fixed :+1: